airbyte-integrations/bases/base-java/run_with_normalization.sh

```diff
@@ -36,6 +36,10 @@ then
 elif test "$NORMALIZATION_TECHNIQUE" = 'LEGACY' && test "$USE_1S1T_FORMAT" != "true"
 then
   echo '{"type": "LOG","log":{"level":"INFO","message":"Starting in-connector normalization"}}'
+  # Normalization tries to create this file from the connector config and crashes if it already exists
+  # so just nuke it and let normalization recreate it.
+  # Use -f to avoid error if it doesn't exist, since it's only created for certain SSL modes.
+  rm -f ca.crt
```

Comment:
Insanity

Reply:
to be fair, destination-postgres is also creating this file (which is why this is only becoming a problem now) https://github.com/airbytehq/airbyte/blob/2d79854279fdf38d70776b57bc003ec9e6a9119e/airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/util/PostgresSslConnectionUtils.java#L139

not that I understand why we need to write a file to begin with :shrug: